SonarQube.Common/Utilities.cs

```diff
@@ -179,6 +179,25 @@ public static bool HandleHostUrlWebException(WebException ex, string hostUrl, IL
 
             return false;
         }
+
+        public static void LogAssemblyVersion(ILogger logger, System.Reflection.Assembly assembly, string description)
+        {
+            if (logger == null)
+            {
+                throw new ArgumentNullException("logger");
+            }
+            if (assembly == null)
+            {
+                throw new ArgumentNullException("asm");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method 'Utilities.LogAssemblyVersion(ILogger, Assembly, string)' passes 'asm' as the 'paramName' argument to a 'ArgumentNullException' constructor. Replace this argument with one of the method's parameter names. Note that the provided parameter name should have the exact casing as declared on the method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AInstantiateArgumentExceptionsCorrectly)